processing/src/main/java/org/apache/druid/query/datasourcemetadata/DataSourceQueryQueryToolChest.java

```diff
@@ -119,4 +121,10 @@ public CacheStrategy getCacheStrategy(DataSourceMetadataQuery query)
   {
     return null;
   }
+
+  @Override
+  public CacheStrategy getCacheStrategy(DataSourceMetadataQuery query, @Nullable ObjectMapper mapper)
+  {
+    return null;
```

Comment:
nit: if this is returning null is the override really needed?

Reply:
ah maybe is safer to leave since default is to not cache I guess? I don't feel super strongly either way